Tasks/DockerV1/containerpush.ts

```diff
@@ -40,6 +40,10 @@ function dockerPush(connection: ContainerConnection, image: string, imageDigestF
 }
 
 export function run(connection: ContainerConnection): any {
+    var imageLsCommand = connection.createCommand();
```

Comment:
Put this in try catch. even this command fails we should ignore.

Reply:
Okay